GettingStartedDeb.md

```diff
@@ -125,3 +125,9 @@ The NuGet.config file should look something like the following:
 </configuration>
 ```
 The important part of this is that you have a package source with aspnetvnext and nuget.org in it.
+
+Nuget restore also checks the hash on the package. So ensure that you have `libcrypto` available. To do this run: 

```

Comment:
Yup, we already tell people to install libssl-dev in our docs: http://docs.asp.net/en/latest/getting-started/installing-on-linux.html